gems/pending/util/miq-password.rb

```diff
@@ -123,7 +123,22 @@ def self.clear_keys
   end
 
   def self.v2_key
-    @v2_key ||= ez_load("#{key_root}/v2_key")
+    @v2_key ||= begin
+      key_file = File.expand_path("v2_key", key_root)
+      if File.exists?(key_file)
+        ez_load(key_file)
+      else
+        msg = <<-EOS
+#{key_file} doesn't exist!
+If you're not a developer, please generate your v2_key using 'appliance_console_cli --key'.
```

Comment:
Ick...not a fan of this being in the miq-password code...it seems weird to have appliance_console stuff on a class that is pretty self-contained.  But I can't think of a better way to notify about this, except to just not do it and use external documentation.